vrecord

```diff
@@ -45,6 +45,7 @@ Usage: ${SCRIPTNAME} [ -e | -r | -p | -a | -x | -n|  -v | -h ] [IDENTIFIER]
       log file, so avoid using this option with 'Visual + Numerical' or any
       playback option that display the log as part of the view.
   -h  display this help menu
+  -s  Generate a test signal and use it in place of external input
```

Comment:
`-s` should be added also in line 27 (Usage) or be moved to the Advanced options

Reply:
good catch!